tests/php/test-class-amp-theme-support.php

```diff
@@ -1547,6 +1559,18 @@ public function test_filter_admin_bar_script_loader_tag( $setup_callback, $asser
 		$assert_callback( new DOMXPath( $dom ) );
 	}
 
+	/**
+	 * Test filter_admin_bar_script_loader_tag when ->deps is not an array.
+	 *
+	 * @covers \AMP_Theme_Support::filter_admin_bar_script_loader_tag()
+	 */
+	public function test_filter_admin_bar_script_loader_tag_non_array() {
+		wp_enqueue_script( 'admin-bar' );
+		$GLOBALS['wp_scripts']->registered['admin-bar']->deps = null;
+		$tag = '<script src="https://example.com/wp-includes/js/admin-bar.js?ver=5.3.2"></script>';
+		$this->assertEquals( $tag, AMP_Theme_Support::filter_admin_bar_script_loader_tag( $tag, 'example' ) );
```

Comment:
I suppose in reality the assertion here serves the purpose of preventing the test from being flagged for having no assertions. Since there is no style registered with the handle of `baz`, neither of the conditions `self::is_exclusively_dependent( wp_scripts(), $handle, 'admin-bar' )` nor `self::has_dependency( wp_scripts(), $handle, 'admin-bar' )` will ever be true.